lib/livebook/hubs/team_client.ex

```diff
@@ -132,14 +144,18 @@ defmodule Livebook.Hubs.TeamClient do
   @spec identity_enabled?(String.t()) :: boolean()
   def identity_enabled?(id) do
     GenServer.call(registry_name(id), :identity_enabled?)
+  catch
+    :exit, _ -> false
   end
```

Comment:
It seems brittle to me to fallback to defaults. Returning an empty lists will likely be fine for the caller, but booleans are less "neutral".

I think ideally we would return `:ok`/`:error` tuple, so that the caller can distinguish the hub being no longer available (now that we treat it as an "expected" state). I would consider the call timing out to be similar to a database error. Typically we wouldn't catch database error and fallback to a certain value, but we would rather propagate the error.

@josevalim wdyt?